### PR TITLE
Lottery

### PR DESCRIPTION
**Pull request by mgsorigo**

### CODE REVIEW

File: src/main/java/core/basesyntax/Lottery.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```
Comment (nowickimj) [inline src/main/java/core/basesyntax/Lottery.java:8]: `colorSupplier` should be a constant

File: src/main/java/core/basesyntax/Lottery.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        String color = colorSupplier.getRandomColor();
+        Random random = new Random();
```
Comment (nowickimj) [inline src/main/java/core/basesyntax/Lottery.java:10]: `random` should be a constant

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        int number = random.nextInt(100) + 1;
```
Comment (nowickimj) [inline src/main/java/core/basesyntax/Lottery.java:11]: Max number should be a constant (e.g. `MAX_NUMBER`)

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 4]
```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        Random random = new Random();
```
Comment (nowickimj) [inline src/main/java/core/basesyntax/ColorSupplier.java:8]: `random` should be a constant

File: src/main/java/core/basesyntax/Lottery.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```
Comment (nowickimj) [inline src/main/java/core/basesyntax/Lottery.java:7]: This method shouldn't be static

File: src/main/java/core/basesyntax/Application.java [hunk 6]
```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier colorSupplier = new ColorSupplier();
```
Comment (nowickimj) [inline src/main/java/core/basesyntax/Application.java:5]: You shouldn't create `ColorSupplier` instance here.
Please read carefully the requirements for this exercise.

**Review by nowickimj — CHANGES_REQUESTED** [review]
Changes required - please check the requirements for this task

**Comment by mgsorigo** [comment]
@nowickimj The changes have been implemented. Request for review.

File: src/main/java/core/basesyntax/Application.java [hunk 7]
```diff
@@ -2,6 +2,17 @@
...
+
+        // Create three balls using the getRandomBall() method
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+
+        // Print information about the balls in the console
+        System.out.println("Ball 1: " + ball1);
+        System.out.println("Ball 2: " + ball2);
+        System.out.println("Ball 3: " + ball3);
```
Comment (nowickimj) [inline src/main/java/core/basesyntax/Application.java:16]: Repetitive tasks should be handled in the loop:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-for-loop-for-creating-several-objects-of-the-same-class

**Review by nowickimj — CHANGES_REQUESTED** [review]
Looks good, one more thing to fix

**Review by nowickimj — APPROVED** [review]

**Pull request closed without merging**